cellxgene_schema_cli/scripts/readme.md

```diff
@@ -95,7 +95,7 @@ To download all these files the following has to be called:
 
 If reference gene versions need to be updated then `Makefile` has to be changed accordingly under the rules corresponding to each organism.
 
-## Updating gene references
+## Cleaning directory
 
 If  instead of the rule `update-references` individual make rules were executed, then make sure to remove all unused downloads using:
```

Comment:
```suggestion
If instead of the rule `update-references` individual make rules were executed, then make sure to remove all unused downloads using:
```